app/Models/ComponentGroup.php

```diff
@@ -11,13 +11,14 @@
 
 namespace CachetHQ\Cachet\Models;
 
+use Illuminate\Support\Collection;
```

Comment:
It shouldn't be in that order, it should be A-Z. StyleCI will be looked in to, seems like a :bug:, thanks :smile: